lib/web3.js

```diff
@@ -121,6 +124,20 @@ class Web3 {
                         amount: info.tokenAmount.uiAmount,
                         type: nft.attributes.itemType,
                       });
+                      switch (nft.name) {
+                        case "Ammunition":
```

Comment:
I think resourceAvailability can actually be removed since I am using inventory for everything now
  inventory.push(...(await refreshInventory()));

if (character?.toString() === "i") {
    return Write.printAvailableSupply(resourceAvailability);
  }
  
  this should pass inventory instead of resourceAvailability

Reply:
I saw that now. yes it should be inventory. but that was not the problem. I do force push that change though.

Reply:
If you want to get rid of resourceAvailability and keep using inventory (which seems to be a better solution than mine)
Then change line 247-249 in start.js
```
if (resourceAvailability[resource] < needed) {
      enoughResources = false;
}
```
as of your last commit this will never get false because resourceAvailability is undefined. You got to get the data from inventory. I am not that firm in javascript. So do not count on me :D